src/awkward/numba.py

```diff
@@ -1,5 +1,9 @@
 # BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
 
+import math
+
+import numpy  # noqa: TID251
```

Comment:
@jpivarski and @agoose77 - has our policy changed? I get the following `ruff` error: 
```
TID251 `numpy` is banned: Use `numpy = ak._nplikes.Numpy.instance()` instead
```